visualize-admin/visualization-tool
# feat: Time slider continuation

This PR continues the work on the `TimeSlider`,
- aligning it with the [design](https://www.notion.so/interactivethings/V02-Timeslider-85eec9497bed42c0a8b762d1e699a59f),
- making it animatable,
- enabling it for more chart types (grouped / stacked column, pie, scatterplot),
- fixing chart scale domains in place.

UI of slider's settings, stepped mode & introduction to maps will be tackled in subsequent PRs.

### Testing
1. Go to `/en/create/new?cube=https://environment.ld.admin.ch/foen/ubd0104/6&flag__timeslider=true`
2. Change X field from `Date` to `Parameter`
3. Enable Time Slider (Interactive Filters)
4. See that the slider appeared and it's possible to animate the chart

### TODO
- [x] Add data option to observable Timeline
